src/FastRoute/src/DBWrapper.cpp

```diff
@@ -427,6 +434,19 @@ void DBWrapper::initNetlist(bool reroute)
   }
 }
 
+void DBWrapper::initClockNets()
+{
+  std::set<odb::dbNet*> _clockNets;
```

Comment:
this should be happening further back in the flow. hopefully this code will go away shortly